src/App.tsx

```diff
@@ -1,24 +1,247 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+
+import classNames from 'classnames';
+import React, {
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import * as todosAPI from './api/todos';
+import { Footer } from './components/Footer';
+import { NewTodoForm } from './components/NewTodoForm';
+import { Loader } from './components/Loader';
+import { TodoList } from './components/TodoList';
+import { Errors, Filter, Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 6101;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
```

Comment:
create enum for filter, string values are really difficult to maintain

Reply:
oh, I see you already have one. Why not just add `Filter.All` for that?

Reply:
Done!